src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -5,8 +5,10 @@
 
 /**
  * Class handling the registering of the mod items.
+ *
+ * We disabled the following finals since we are neither able to mark the items as final, nor do we want to provide public accessors.
  */
-@SuppressWarnings("squid:S1444")
+@SuppressWarnings({"squid:ClassVariableVisibilityCheck", "squid:S2444", "squid:S1444"})
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:ClassVariableVisibilityCheck, squid:S2444, squid:S1444' warnings is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1309)